src/views/alpha/FeedCard.tsx

```diff
@@ -247,7 +247,15 @@ export const ProfileHandleStack = ({ users }: { users: User[] }) => {
   );
 };
 
-export const ProfileHandle = ({ user, shorten = false }: { user: User; shorten?: boolean }) => {
+export const ProfileHandle = ({
+  user,
+  shorten = false,
+  ...props
+}: {
+  user: User;
+  shorten?: boolean;
+  popoverPlacement?: 'top' | 'bottom';
+}) => {
   const [twitterHandle, setTwitterHandle] = useState(user.profile?.handle);
   const [twitterHandleQuery, twitterHandleQueryContext] = useTwitterHandleFromPubKeyLazyQuery();
```

Comment:
Why does each feedcard need to fetch the profile? That is a lot of requests. The profile info is available of the feed event query better to consume from there.

Reply:
It's a lazy query so it's only called when the required info is not available, but looking into removing it now